# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/srecord/hotfix-cstdint.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Include cstdint for uint16_t, no longer pulled in indirectly with GCC 13+.

--- srecord-1.65.0-Source.orig/srecord/input/file/hp64k.h	2026-09-27 13:18:38.543201769 +0200
+++ srecord-1.65.0-Source/srecord/input/file/hp64k.h	2026-09-27 13:18:38.549828535 +0200
@@ -20,6 +20,7 @@
 #ifndef LIB_INPUT_FILE_HP64K
 #define LIB_INPUT_FILE_HP64K
 
+#include <cstdint>
 #include <srecord/input/file.h>
 
 namespace srecord {
